README.md

```diff
@@ -202,6 +208,14 @@ apt::sources:
 * `settings`: Passes a hash to `create\_resource` to make new `apt::setting` resources.
 * `sources`: Passes a hash to `create\_resource` to make new `apt::source` resources.
 
+####apt::backports
+
+* `location`: The URL of the apt repository. OS-dependent defaults are specifed in `apt::params` for Ubuntu and Debian. Required parameter for other OSes.
+* `release`: The distribution of the apt repository. Defaults to "${lsbdistcodename}-backports" for Ubuntu and Debian. Required parameter for other OSes.
+* `repos`: The component of the apt repository. OS-dependent defaults are speicifed in `apt::params` for Ubuntu and Debian. Required parameter for other OSes.
+* `key`: The key for the backports repository. Can either be a string or a hash. See apt::setting for details on passing key as a hash. OS-dependent defaults are specified in `apt::params` for Ubuntu and Debian. Required parameter for other OSes.
+* `pin`: The pin priority for backports repository. Can either be a number, a string, or a hash that will be passed to `create\_resource to make a new pin. Defaults to `200`.
```

Comment:
I wouldn't mention the implementation detail of `create_resources`, just that it will be passed as parameters to `apt::pin`.